test/fixtures/reports.yml

```
@@ -1,9 +1,10 @@
-report1:
-  title: Test Report 1
-  content: This is a test report 1
-  user: one
+john_report:
+  title: john Report
+  content: This is a john report
```

プログラミングとは無関係ですが、

```suggestion
  content: This is John's report
```

が英語的に正ですかね〜。